internal/gengapic/testdata/lro_client_init.want

```diff
@@ -48,15 +70,18 @@ func NewFooClient(ctx context.Context, opts ...option.ClientOption) (*FooClient,
 	if err != nil {
 		return nil, err
 	}
-	c := &FooClient{
+	client := FooClient{CallOptions: defaultFooCallOptions()}
+
+	c := &fooGrpcClient{
 		connPool:    connPool,
 		disableDeadlines: disableDeadlines,
-		CallOptions: defaultFooCallOptions(),
-
 		fooClient: mypackagepb.NewFooClient(connPool),
+		CallOptions: &client.CallOptions,
 	}
 	c.setGoogleClientInfo()
 
+	client.internalFooClient = c
+
 	c.LROClient, err = lroauto.NewOperationsClient(ctx, gtransport.WithConnPool(connPool))
```

Comment:
I mentioned this on the gapic-showcase draft PR, but I think the `LROClient` needs to be handled the same way as the `CallOptions` with the double pointer situation - right? If we expose LROClient on the wrapper, it should be initialized and set there (on the wrapper) and _referenced_ in the transport-client

Reply:
Done

Reply:
Done